cronjobs/pootle-importer

```diff
@@ -9,14 +9,15 @@ MAILTO=translate-admin@mycroft.ai
 set -e
 
 # output to a log file and if there are errors, send email
-if ! ~/mycroft-translate/automations/importer.sh 2>&1 > /tmp/pootle-importer.log; then
+if ! /root/mycroft-translate/automations/importer.sh 2>&1 > /tmp/pootle-importer.log; then
 	#send email
 	(
 		echo "To:$MAILTO"
 		echo "Subject: Pootle Importer Daily Job failed"
 		echo "MIME-Version: 1.0"
 		echo "Content-Type: text/plain"
 		echo cat /tmp/pootle-importer.log
+                echo "\nHOST: " $(hostname)
```

Comment:
Minor nit-pick here. the other lines seem to be indented with tabs while this is with spaces.